libs/core-ui/src/lib/util/getFeatureNamesAfterDrop.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
```

Comment:
nit: Maybe name of this file could be more generic, like featureNameUtils or datasetUtils, and we can put related methods in same file?  I think that might be a better way to organize than to have a single file per method?

Reply:
Updated!